Sources/KSCrashRecordingCore/include/KSCrashMonitorHelper.h

```diff
@@ -0,0 +1,72 @@
+//
+//  KSCrashMonitorHelper.h
+//
+//
+//  Created by Gleb Linnik on 03.06.2024.
+//
+
+#ifndef KSCrashMonitorHelper_h
+#define KSCrashMonitorHelper_h
+
+#include "KSCrashMonitor.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+static inline void kscm_setMonitorEnabled(KSCrashMonitorAPI* api, bool isEnabled)
```

Comment:
nit: not sure, but looks like these functions are good with `const`s
```suggestion
static inline void kscm_setMonitorEnabled(const KSCrashMonitorAPI* api, bool isEnabled)
```